webpack/ForemanInventoryUpload/Components/PageHeader/components/CloudConnectorButton/CloudConnectorButton.js

```diff
@@ -10,26 +10,26 @@ export const CloudConnectorButton = ({ status, onClick, jobLink }) => {
       <Popover
         bodyContent={
           <div>
-            {__('Cloud connector job is still running, you can view it here:')}
+            {__('Cloud connector setup is in progress now: ')}
             <a href={jobLink} target="_blank" rel="noopener noreferrer">
-              {__('Open job')}
+              {__('Cloud connector job link')}
             </a>
           </div>
         }
         aria-label="Popover with Link to cloud connector job"
         closeBtnAriaLabel="Close cloud connector Popover"
       >
         <Button variant="secondary">
-          <Spinner size="sm" /> {__('Cloud Connector in progress')}
+          <Spinner size="sm" /> {__('Cloud Connector is in progress')}
         </Button>
       </Popover>
     );
   }
 
   if (status === CONNECTOR_STATUS.RESOLVED) {
     return (
-      <Button variant="secondary" isDisabled>
-        {__('Cloud Connector is configured')}
+      <Button variant="secondary" onClick={onClick}>
+        {__('Reconfigure Cloud Connector')}
```

Comment:
Can we add an Alert for safety, asking 
`"Cloud connector is already set, do you really want to reconfigure it?"`

Reply:
I don't think it's necessary. `Reconfigure Cloud Connector` should be enough

Reply:
IMO some users can read it and get the feeling that something is wrong with the current setup, but maybe it's just me :)